docs/src/Table.doc.js

```diff
@@ -333,6 +333,14 @@ card(
         type: 'string',
         required: true,
       },
+      {
+        name: 'onExpand',
+        type: `({ event: SyntheticMouseEvent<HTMLButtonElement> | SyntheticKeyboardEvent<HTMLButtonElement> | SyntheticMouseEvent<HTMLAnchorElement>
+          | SyntheticKeyboardEvent<HTMLAnchorElement> expanded: boolean }) => void`,
+        description: [
+          'Callback fired when the expand arrow-down button component is clicked with a mouse or keyboard.',
```

Comment:
@faisalgedi could we update this to

> 'Callback fired when the expand button component is clicked.'

* We also do a callback with the `arrow-up` button
* We can't technically 'click' with a keyboard.